sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -320,9 +319,6 @@ private[hive] class SparkSQLCLIDriver extends CliDriver with Logging {
   // because the Hive unit tests do not go through the main() code path.
   if (!isRemoteMode) {
     SparkSQLEnv.init()
-    if (sessionState.getIsSilent) {
-      SparkSQLEnv.sparkContext.setLogLevel(Level.WARN.toString)
```

Comment:
@wangyum, I was just double checking. Shouldn't we still respect this option when the log level of this class is explicitly set? It's a bit unlikely but I just wanted to check for doubly sure.

Reply:
Could you share an example?

Reply:
For example,

```bash
cd conf
echo "log4j.logger.org.apache.spark.sql.hive.thriftserver.SparkSQLCLIDriver=INFO" >> log4j.properties.template
mv log4j.properties.template log4j.properties
cd ..
./bin/spark-sql -S
```
```
20/09/09 14:58:27 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
20/09/09 14:58:27 INFO HiveConf: Found configuration file null
20/09/09 14:58:28 INFO SharedState: spark.sql.warehouse.dir is not set, but hive.metastore.warehouse.dir is set. Setting spark.sql.warehouse.dir to the value of hive.metastore.warehouse.dir ('/user/hive/warehouse').
20/09/09 14:58:28 INFO SharedState: Warehouse path is '/user/hive/warehouse'.
...
```

Reply:
Hmm, silent mode should be given preference over the static configuration.